_posts/2020-11-30-developer-survey-2020.md

```diff
@@ -0,0 +1,99 @@
+
+![Frontend languages](/img/developer-survey-2020/frontend_languages.png)
+
+For frontend languages the clear number one was JavaScript with 68% usage. The other languages that have wide usage are Typescript (37%) and ClojureScript (19%). ClojureScript is an obvious choice for projects that use Clojure in the backend, so it seems those two go hand in hand quite much. CoffeeScript has a handful of users - it seems that the number has been stable compared to 2018 figures (4% overall usage in 2020 compared to 5% in 2018). Most likely it's not used for any new projects but it is still in use in some long-time project(s). PureScript also had a handful users here, which seems to be new compared to 2018.
+
+## Languages we like
+![Liked languages](/img/developer-survey-2020/liked_languages.png)
+
+Next, we asked for opinions: what language do you like the most? The most used backend language was also the most liked one, as 17% of respondents said that Java was their favorite language. Clojure came second with 15%, which is impressive considering that even though it is used quite much, in terms of raw numbers Java developers outnumber Clojure developers by a mile. Clojure developers then really know what they want. Kotlin was the third most liked language, meaning that the top 3 of liked languages mirrors the top 3 of backend languages by usage.
+
+Compared to results from 2018, JavaScript fell down from number #2 spot to number #4. This could be explained TypeScript, which gained almost as many percents now than JavaScript lost. All in all, most top languages were in decline compared to the previous survey, except Kotlin, which received much more love this year compared to 2018. This time there also seemed to be a lot more niche languages getting a few votes than before, which also explains why the top languages were a bit in decline.
+
+## Languages we would like to use
+![Languages we would like to use](/img/developer-survey-2020/would_like_to_use_languages.png)
+
+The answers to this question tells us several things. First, it tells us that for widely already used languages there is still interest in continuing to use them. Second, the answers can tell us that there's even more interest in some languages, so their usage could still grow in the future. Third, for languages that are not yet widely used, the answers tell us what languages we could be potentially be using in the future.
+
+Kotlin was the most wanted language here, with 29% of the respondents saying they would like to use Kotlin in the future. As such, people using Kotlin currently seem to want to continue using it and others would like to start using it. As such, Kotlin seems like a good choice for future projects.
+
+Clojure came third, with 22% wanting to work with it in future. TypeScript came third with 18%. With TypeScript usage having increased a lot in two years, the interest is still growing.
```

Comment:
Clojure and TypeScript both came third place - maybe there is an error?

Reply:
Fixed, Clojure was second and TypeScript third.